DisCatSharp/Entities/Guild/DiscordGuild.cs

```diff
@@ -1593,6 +1593,14 @@ public async Task<IReadOnlyList<DiscordAuditLogEntry>> GetAuditLogsAsync(int? li
                 });
             }
 
+            var atgse = alrs.SelectMany(xa => xa.ScheduledEvents)
```

Comment:
whatever alrs is. or atgse for that matter

Reply:
for later, audit log is cursed